versioning/BRM/extentmap.cpp

```diff
@@ -1941,7 +1953,11 @@ void ExtentMap::grabEMEntryTable(OPS op)
     }
     else
     {
-      fPExtMapRBTreeImpl = ExtentMapRBTreeImpl::makeExtentMapRBTreeImpl(fEMRBTreeShminfo->tableShmkey, 0);
+      const bool isReadOnly = false;
+      // The ternary sends EM segment table only if this grab is for a read to allow to upgrade the RWLock if
+      // case of a shmem segment remap.
+      fPExtMapRBTreeImpl = ExtentMapRBTreeImpl::makeExtentMapRBTreeImpl(
+          fEMRBTreeShminfo->tableShmkey, 0, isReadOnly, emLocked, op == READ ? &fMST : nullptr);
```

Comment:
Can you please remove the var `isReadOnly` and directly pass `false` to the function.

Also, a minor grammar error in the comment: "if case of a shmem" should be replaced by "in case of a shmem".

Reply:
Changed the comment wording though.